### DISCUSSION TITLE
Latest dev build can't find the data and music files.

### DISCUSSION THREAD

**Discussion by eMonolith** [ihhub/fheroes2#9311] [Q&A]
I just compiled and installed the latest dev build and then installed both DEB files, data, and music. installed original game files. I logged out and back in and started the game, but i seems to not find the data and music files. I am running ubuntu numbat.

**Comment by ihhub** [comment]
Hi @eMonolith , please provide exact steps (commands) you did in order to compile and install the latest dev build. What is the path you "install" the engine? What is the path of data and music directories?

**Reply by eMonolith** [reply]
I installed the 1.1.4 version deb file from Ubuntu. Then I installed the data and music deb files from my build on my laptop.

**Reply by eMonolith** [reply]
I am trying to install to my desktop.

**Reply by eMonolith** [reply]
For some reason I had an fheroes2 file in /usr/bin. I deleted it, reinstalled everything, and now my desktop has fheroes2. Thank you for keeping the game alive.

**Comment by armorder** [comment]
У меня такая же шляпа на винде происходит. fheroes2 не подхватывают ресы от оригинала. Пробовал переустанавливать несколько раз, но результат тот же. Возможно проблема в том, что изначально я ставил движок с демоверсией?

**Reply by oleg-derevenetz** [reply]
А где вообще ресурсы оригинальной игры у вас находятся-то? Есть ли они в каталоге, в который установлена игра, или в каталоге `%APPDATA%\fheroes2` (это два места, в которых игра будет их искать)? Там должны быть как минимум каталоги `DATA` и `MAPS`, плюс еще могут быть `ANIM` и `MUSIC`.

**Reply by armorder** [reply]
я, согласно readme, запускал extract_homm2_resources.bat, вводил там адрес папки с оригиналом (C:\HOMM2) и далее скрипт выдавал "press any key...", никаких ошибок не было. Из чего  я делал вывод, что процедура копирования прошла нормально. Я пробовал и руками копировать файлы в папку C:\Program Files\fheroes2\files\data, но тоже безрезультатно.

**Reply by oleg-derevenetz** [reply]
> Я пробовал и руками копировать файлы в папку C:\Program Files\fheroes2\files\data, но тоже безрезультатно.

В эту папку ничего копировать не нужно, она предназначена для другого. Ресурсы оригинальной игры (папки `DATA`, `MAPS`, и так далее) должны находиться либо в каталоге установки игры (в вашем случае - в `C:\Program Files\fheroes2`, *непосредственно в этом каталоге*), либо в пользовательском каталоге (`%APPDATA%\fheroes2`, туда можно попасть через ярлык "Game data files" в группе "fheroes2" в меню Пуск). Убедитесь, что не получилось так, что, скажем, в пользовательском каталоге у вас лежат ресурсы демо-версии, а в каталоге установки игры - ресурсы полной версии оригинальной игры.

**Reply by armorder** [reply]
Попробовал полностью удалить игру, со всеми папками, перезагрузил комп, установил движок, закинул ресы оригинала в папку fHeroes2, запустил скрипт и результат тот же, игра ресы не видит. Не знаю, может я не ту версию оригинала использую (у меня Price of loyalty под DOS). И что значит полная версия? Как отличить полную от неполной?

**Reply by oleg-derevenetz** [reply]
> Не знаю, может я не ту версию оригинала использую (у меня Price of loyalty под DOS)

В `%APPDATA%/fheroes2` должен лежать файл `fheroes2.log, что в нем есть? Запостите его содержимое сюда, пожалуйста. Ещё такой момент - fheroes2 может не работать с некоторыми пиратскими "локализованными" версиями оригинальной игры из-за ряда проблем с ними (например с генерацией шрифтов).

> И что значит полная версия? Как отличить полную от неполной?

Полная версия - это значит не демо-версия (та, которая Succession Wars без кампании и с одной картой, распространялась бесплатно).

**Reply by armorder** [reply]
Всё, спасибо большое за подсказки, я разобрался. Действительно это были не полные ресы. Достал полный оригинал и с ним  всё заработало.
